kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -768,6 +752,29 @@ def foundry_step_node(
             apr_proof.write_proof_data()
 
 
+def foundry_get_apr_proof(
+    foundry: Foundry,
+    test: str,
+    bug_report: bool = False,
+) -> APRProof:
+    proof = foundry_get_proof(foundry=foundry, test=test, bug_report=bug_report)
+    if not isinstance(proof, APRProof):
+        raise ValueError('Specified proof is not an APRProof.')
+    return proof
+
+
+def foundry_get_proof(
+    foundry: Foundry,
+    test: str,
+    bug_report: bool = False,
+) -> Proof:
+    proofs_dir = foundry.out / 'apr_proofs'
+    contract_name, test_name = test.split('.')
+    proof_digest = foundry.proof_digest(contract_name, test_name)
+    proof = Proof.read_proof_data(proofs_dir, proof_digest)
+    return proof
+
+
```

Comment:
Because these are both methods that take `Foundry` as the first argument, they might as well become methods in the class.

```
class Foundry:

    def get_apr_proof(test: str) -> APRProof:
        ...

    def get_proof(test: str) -> Proof
        ...
```

Also, as far as I can tell, neither needs tho `BugReport` option anymore.

Reply:
Done.